template/templates/views/categories.html

```diff
@@ -20,7 +20,7 @@ <h1>{{ t "page.categories.title" }} ({{ .total }})</h1>
                 <span class="item-title">
                     <a href="{{ route "categoryEntries" "categoryID" .ID }}">{{ .Title }}</a>
                 </span>
-                (<span title="{{ if eq .FeedCount 0 }}{{ t "page.categories.no_feed" }}{{ else }}{{ plural "page.categories.feed_count" .FeedCount .FeedCount }}{{ end }}">{{ .FeedCount }}</span>)
+                [<span title="{{ t "page.categories.unread_counter" }}">{{ .TotalUnread }}</span>]
```

Comment:
Can you please keep the parenthesis instead of using brackets? It's just matter of consistency with the rest.

Reply:
sure! had just thought it might be a good idea to visually differentiate because the method of counting changed. reverted them back to parantheses now